src/renderer/screens/Operations/common/utils.ts

```diff
@@ -219,3 +222,23 @@ export const getTransactionAmount = (tx: Transaction): string | null => {
 
   return null;
 };
+
+export const getSignatoryName = (
+  signatoryId: AccountId,
+  tx: MultisigTransaction,
```

Comment:
If we only get `signatories` from `tx` then it's preferably to pass `tx.signatories`
So the argument would become - `signatories: Signatory[]`